businesstime/australia.py

```diff
@@ -0,0 +1,101 @@
+
+from datetime import date
+from businesstime import Holidays
+
+class QueenslandPublicHolidays(Holidays):
+    """
+    http://www.qld.gov.au/recreation/travel/holidays/public/
+
+    Current coverage is only 2013-2016.
+    """
+
+    # Australian public holidays are legislated by each state individually for 
```

Comment:
And in case that sounds like a theoretical concern, our state government has already flipped Queen's Birthday and Labour Day to opposite ends of the year twice since 2011. :-)